addons/l10n_in/views/report_invoice.xml

```diff
@@ -10,12 +10,7 @@
         </xpath>
         <xpath expr="//t[@t-set='address']" position="inside">
             <t t-if="o.company_id.account_fiscal_country_id.code == 'IN' and o.l10n_in_state_id" class="mt16">
-                <t t-if="o.move_type in ('in_invoice', 'in_refund')">
```

Comment:
@jva-odoo Destination of supply was a wrong concept on vendor bills then?

Reply:
yes, in both cases it's call Place of supply.